Sources/StreamChat/Controllers/PollController/PollVoteListController.swift

```diff
@@ -126,6 +127,7 @@ public class PollVoteListController: DataController, DelegateCallable, DataStore
 
         pollsRepository.queryPollVotes(query: query) { [weak self] result in
             guard let self else { return }
+            self.votesCursor = result.value?.next
```

Comment:
This is also why the `hasLoadedAllVotes: Bool { votesCursor == nil }` is a bad idea. If the response is an error, there will be no cursor, but it will say we loaded all votes already